### PR TITLE
Feature/validation registration

### PR DESCRIPTION
**Pull request by lhalam**
## JIRA

* [Main JIRA ticket](https://ssu-jira.softserveinc.com/secure/RapidBoard.jspa?rapidView=738)

## Code reviewers

- [ ] @dmytrorevak
- [ ] @IvanAndrukh
- [ ] @JjuliaP
- [ ] @makarman
- [ ] @noobhunters
- [ ] @Silent777
- [ ] @Staryj812

### Second Level Review

- [ ] @ssOleg
- [x] @lhalam

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: eventually/static/src/containers/registerLogin/Forget.js [hunk 1]
```diff
@@ -3,23 +3,40 @@ import {withRouter} from 'react-router-dom';
 import TextField from 'material-ui/TextField';
 import RaisedButton from 'material-ui/RaisedButton';
 import {forgetPasswordService} from './registrationService.js';
+import {orange500, blue500, green700} from 'material-ui/styles/colors';
 
 const style = {
     margin: 12,
 };
+const styles = {
```
Comment (lhalam) [inline eventually/static/src/containers/registerLogin/Forget.js]:
```
const errorStyle = {
    color: orange500,
 };
```

File: eventually/static/src/containers/registerLogin/Login.js [hunk 2]
```diff
@@ -3,27 +3,54 @@ import {withRouter} from 'react-router-dom';
...
-        this.setState({email: event.target.value});
-    };
 
-    handlePassword = event => {
-        this.setState({password: event.target.value});
+        const regex = /^\S+@\S+\.\S+$/;
+        if(regex.test(event.target.value) === true )
+        {
+            this.setState({MessageEmail: '', email: event.target.value});
+
```
Comment (lhalam) [inline eventually/static/src/containers/registerLogin/Login.js]: remove empty line

File: eventually/static/src/containers/registerLogin/Register.js [hunk 3]
```diff
@@ -3,9 +3,25 @@ import {withRouter} from 'react-router-dom';
 import TextField from 'material-ui/TextField';
 import RaisedButton from 'material-ui/RaisedButton';
 import {registerService} from './registrationService.js';
+import {orange500, blue500, green700,pink600} from 'material-ui/styles/colors';
 
 const style = {
     margin: 12,
+
```
Comment (lhalam) [inline eventually/static/src/containers/registerLogin/Register.js]: remove empty lines

File: eventually/static/src/containers/registerLogin/Register.js [hunk 4]
```diff
@@ -3,9 +3,25 @@ import {withRouter} from 'react-router-dom';
...
+import {orange500, blue500, green700,pink600} from 'material-ui/styles/colors';
 
 const style = {
     margin: 12,
+    
+
+};
+
+const styles = {
+
```
Comment (lhalam) [inline eventually/static/src/containers/registerLogin/Register.js]: remove empty lines

File: eventually/static/src/containers/registerLogin/Register.js [hunk 5]
```diff
@@ -35,19 +73,30 @@ class Register extends React.Component {
...
     render() {
         return (
             <div style={style}>
                 <h2>Email</h2>
                 <TextField
                     onChange={this.handleEmail}
                     hintText='email'
+                    errorText={this.state.MessageEmail}
+                    errorStyle={styles.errorStyle}
+
```
Comment (lhalam) [inline eventually/static/src/containers/registerLogin/Register.js]: remove empty line

File: eventually/static/src/containers/registerLogin/Register.js [hunk 6]
```diff
@@ -35,19 +73,30 @@ class Register extends React.Component {
         event.preventDefault();
     };
 
+
```
Comment (lhalam) [inline eventually/static/src/containers/registerLogin/Register.js]: remove empty line

File: eventually/static/src/containers/registerLogin/Register.js [hunk 7]
```diff
@@ -35,19 +73,30 @@ class Register extends React.Component {
...
+                    errorStyle={styles.errorStyle}
+                    
+                    type="email"
                 /><br/>
                 <br/>
                 <h2>Password</h2>
                 <TextField
                     onChange={this.handlePassword}
                     hintText='password'
+
```
Comment (lhalam) [inline eventually/static/src/containers/registerLogin/Register.js]: remove empty line

File: eventually/static/src/containers/registerLogin/Register.js [hunk 8]
```diff
@@ -35,19 +73,30 @@ class Register extends React.Component {
...
                 /><br/>
                 <br/>
                 <h2>Password</h2>
                 <TextField
                     onChange={this.handlePassword}
                     hintText='password'
+                    
+                    errorText={this.state.MessagePassword}
+                    errorStyle={styles.errorStyle}
+
```
Comment (lhalam) [inline eventually/static/src/containers/registerLogin/Register.js]: remove empty line

**Review by VitalyiKhlivnyi — APPROVED** [review]

**Pull request merged**